tests/Doctrine/Tests/DBAL/Schema/OracleSchemaManagerTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Schema;
+
+use Doctrine\DBAL\Schema\Table;
+use PHPUnit\Framework\TestCase;
+
+class OracleSchemaManagerTest extends TestCase
```

Comment:
It needs to be an integration test that would call `listTableDetails()` and fail on Oracle thereby reproducing the problem. Unit tests are not really helpful in this case.

Reply:
I get your point, you're right. But I don't have any experience doing functional tests. Could you point me to a similar example within the project to use as a guide? Thank you very much!

Reply:
Look at the [`DateExpressionTest`](https://github.com/doctrine/dbal/blob/v2.10.1/tests/Doctrine/Tests/DBAL/Functional/Platform/DateExpressionTest.php) or [`DataAccessTest`](https://github.com/doctrine/dbal/blob/v2.10.1/tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php) for example.

Mind that the `master` branch currently represents DBAL 4.0 with the unknown release date. Since this is a bug fix, you will most likely want to target your PR against the `2.10.x` branch.